### PR TITLE
EL-2663: Schema for format dropcaps

### PR DESCRIPTION
**Pull request by michaelguild13**
Format, dropcap schema based on PR
https://github.com/washingtonpost/ans-schema/pull/200

### CODE REVIEW

**Comment by gengel** [comment]
I think this needs to be against at least 0.10.1 now, better even 0.10.2

Also, tests need to be added or altered

**Comment by michaelguild13** [comment]
> I think this needs to be against at least 0.10.1 now, better even 0.10.2
> 
> Also, tests need to be added or altered

Ok, I'll update to 0.10.2 and check tests

File: src/main/resources/schema/ans/0.10.0/story_elements/header.json [hunk 1]
```diff
@@ -21,7 +21,9 @@
     "alignment": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_alignment.json"
     },
-
+    "format": {
```
Comment (yury-sannikov) [inline src/main/resources/schema/ans/0.10.0/story_elements/header.json]: I think header should not have this element

File: src/main/resources/schema/ans/0.10.0/story_elements/text.json [hunk 2]
```diff
@@ -21,6 +21,9 @@
     "alignment": {
       "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.0/traits/trait_alignment.json"
     },
+    "format": {
```
Comment (yury-sannikov) [inline src/main/resources/schema/ans/0.10.0/story_elements/text.json]: @gengel do you think it make sense to call it format? Maybe something like `block_properties`. Format kind of confusing with text format.

Comment (yury-sannikov) [reply src/main/resources/schema/ans/0.10.0/story_elements/text.json]: Also, file name traits/trait_format.json sounds cryptic to me

File: src/main/resources/schema/ans/0.10.0/story_elements/header.json (hunk 1 above)
Comment (michaelguild13) [reply src/main/resources/schema/ans/0.10.0/story_elements/header.json]: I was originally discussed to have it but I am fine with removing that as well

File: src/main/resources/schema/ans/0.10.0/story_elements/text.json (hunk 2 above)
Comment (michaelguild13) [reply src/main/resources/schema/ans/0.10.0/story_elements/text.json]: Agreed, just following everyone's file naming convention

Comment (michaelguild13) [reply src/main/resources/schema/ans/0.10.0/story_elements/text.json]: `block_properties` works...or `formatting`? I am open. This probably should've been more covered during proposal

**Comment by michaelguild13** [comment]
Opened a new PR that targets 0.10.2 with test story
https://github.com/washingtonpost/ans-schema/pull/212

**Pull request closed without merging**